### PR TITLE
Add doc for check-inject

### PR DESCRIPTION
**Pull request by hanxiaop**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Ambient
- [ ] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [x] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/en/docs/ops/diagnostic-tools/check-inject/index.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+---
+title: Verifying Istio Sidecar Injection with Istioctl Check-Inject
+description: Learn how to use istioctl check-inject to confirm if Istio sidecar injection is properly enabled for your deployments.
+weight: 45
+keywords: [istioctl, injection, kubernetes]
+owner: istio/wg-user-experience-maintainers
+test: no
+---
+
+`istioctl experimental check-inject` is a diagnostic tool that helps you verify if Istio sidecar injection will occur for a specific webhook in your pods. Use this tool to check if the sidecar injection configuration is correctly applied to a live cluster.
```
Comment (frankbu) [inline content/en/docs/ops/diagnostic-tools/check-inject/index.md:10]:
```suggestion
`istioctl experimental check-inject` is a diagnostic tool that helps you verify if specific webooks will perform Istio sidecar injection in your pods. Use this tool to check if the sidecar injection configuration is correctly applied to a live cluster.
```

File: content/en/docs/ops/diagnostic-tools/check-inject/index.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+## Quick Start
+
+To check if Istio sidecar injection did/didn't(or will/won't) occur for a specific pod, run:
```
Comment (frankbu) [inline content/en/docs/ops/diagnostic-tools/check-inject/index.md:14]:
```suggestion
To check why Istio sidecar injection did/didn't (or will/won't) occur for a specific pod, run:
```

File: content/en/docs/ops/diagnostic-tools/check-inject/index.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+For example, if you have a deployment named `httpbin` in the `hello` namespace and a pod named `httpbin-1234` with the label `app=httpbin`, the following commands are equivalent:
+
+{{< text syntax=bash >}}
+$ istioctl experimental check-inject -n hello httpbin-1234
+
+$ istioctl experimental check-inject -n hello deploy/httpbin
+
+$ istioctl experimental check-inject -n hello -l app=httpbin
+{{< /text >}}
```
[the remark this answers is not in the document]
Comment (hanxiaop) [reply content/en/docs/ops/diagnostic-tools/check-inject/index.md]: @frankbu Thanks for the review!

**Review by frankbu — APPROVED** [review]
LGTM

**Pull request merged**